canine/utils.py

```diff
@@ -197,6 +198,28 @@ def get_default_gcp_project():
         )
     return __DEFAULT_GCP_PROJECT__
 
+__DEFAULT_GCP_USER__ = None
+
+def get_gcp_username():
+    """
+    Returns the current application-default username
+    """
+    global __DEFAULT_GCP_USER__
+    try:
+        if __DEFAULT_GCP_USER__ is None:
+            creds, _ = google.auth.default()
```

Comment:
Do we need this line if `creds` is being passed directly to `AuthorizedSession`?

Reply:
Nope! Don't know what happened here